app/helpers/role_organisations_helper.rb

```diff
@@ -1,6 +1,6 @@
 module RoleOrganisationsHelper
   def options_for_your_organisation_select(current_user)
-    organisations = Pundit.policy_scope(current_user, Organisation).not_closed.order(:name)
```

Comment:
I'm probably overthinking this, but removing this policy makes me _slightly_ uneasy - because this is a helper method (and `users_helper` too), the caller has to know whether or not it's acceptable to call this method and render all organisations, e.g if this were called outside of the context of the protected controller methods, so maybe having that double authentication is safer than not having it.

That said, we can probably revisit this/put it back in if we were to ever allow non `govuk_admins` to select organisations for any reason.

Reply:
Yeah I was a little uneasy about that too. I've reinstated the scope for now, but also added a note in the commit message about another method elsewhere in the codebase that makes things even more confusing!

Reply:
That's this commit message, where I also explained the choice to retain the policy scope: https://github.com/alphagov/signon/pull/3178/commits/8075a729eeb8fe34da5039e55ffa7eb7914d6d51

Reply:
Nice one, sounds good to me.